### PR TITLE
Update colour.py

### PR DESCRIPTION
**Pull request by AryamanSrii**
added two new colors to provide users desirable colors

### CODE REVIEW

File: discord/colour.py [hunk 1]
```diff
@@ -324,6 +324,25 @@ def yellow(cls: Type[CT]) -> CT:
...
         return cls(0xFEE75C)
+    
+  
+    @classmethod
+    def cyan(cls: Type[CT]) -> CT:
+        """A factory method that returns a :class:`Colour` with a value of ``#33cccc``.
+
+        .. versionadded:: 2.0
+        """
+        return cls(#33cccc)
```
Comment (Tweak4141) [inline discord/colour.py:335]:
```suggestion
        return cls(0x33cccc)
```

File: discord/colour.py [hunk 2]
```diff
@@ -324,6 +324,25 @@ def yellow(cls: Type[CT]) -> CT:
...
+            
+    
+    @classmethod
+    def light_gold(cls: Type[CT]) -> CT:
+        """A factory method that returns a :class:`Colour` with a value of ``#ff872``.
+
+        .. versionadded:: 2.0
+        """
+        return cls(#ff872)
```
Comment (Tweak4141) [inline discord/colour.py:344]:
```suggestion
        return cls(0xFF872)
```

**Review by Tweak4141 — CHANGES_REQUESTED** [review]
Python comments.

**Review by Tweak4141 — APPROVED** [review]

**Comment by Seniatical** [comment]
Looks good

**Pull request merged**